openapi/endpoints/cards/models/card-order-request.yaml

```diff
@@ -0,0 +1,20 @@
+type: object
+required:
+  - transactionHash
+  - amount
+  - cardProgramId
```

Comment:
isn't it optional?

Reply:
It's only optional for backwards compatability. It will be required in the endpoint at some point